cmd/controller-manager/app/controller-manager.go

```diff
@@ -303,7 +303,12 @@ func setOptionsByFederationConfig(opts *options.Options) {
 	if fedConfig == nil {
 		// FederationConfig could not be sourced from --federation-config or from the API.
 		// create a new `FederationConfig` with default values.
-		fedConfig = &corev1a1.FederationConfig{}
+		fedConfig = &corev1a1.FederationConfig{
```

Comment:
Please add a log message saying that the controller is creating a `FederationConfig` with default settings at this point

Reply:
Done!